src/LibVLCSharp.Forms/Shared/PlaybackControls.xaml.cs

```diff
@@ -821,14 +859,126 @@ private void OnAudioTracksChanged(object sender, EventArgs e)
+            if (AudioTracksListView != null && VideoTracksListView != null && SubtitlesTracksListView != null)
+            {
+                AudioTracksListView.ItemsSource = audioTrackSouces;
+                VideoTracksListView.ItemsSource = videoTrackSouces;
+                SubtitlesTracksListView.ItemsSource = subtitleTracksSources;
+            }
+
+            if (TracksOverLayView != null)
+                TracksOverLayView.IsVisible = true;
+        }
+
+        private void AudioTracksItemTapped(object sender, ItemTappedEventArgs e)
+        {
+            var track = (TrackDescription)e.Item;
+            var manager = Manager.Get<AudioTracksManager>();
+            SelectTrack(manager, track, AudioTracksListView);
+        }
+
+        private void VideoTracksItemTapped(object sender, ItemTappedEventArgs e)
         {
-            await SelectTrackAsync(Manager.Get<AudioTracksManager>(), ResourceManager.GetString(nameof(Strings.AudioTracks)));
+            var track = (TrackDescription)e.Item;
+            var manager = Manager.Get<VideoTracksManager>();
+            SelectTrack(manager, track, VideoTracksListView);
+        }
+
+        private void SubtitlesTracksItemTapped(object sender, ItemTappedEventArgs e)
+        {
+            var track = (TrackDescription)e.Item;
+            var manager = Manager.Get<SubtitlesTracksManager>();
+            SelectTrack(manager, track, SubtitlesTracksListView);
+        }
+
+        private void SelectTrack(TracksManager manager, TrackDescription track, ListView? tracksListview)
+        {
+            var tracks = manager.Tracks;
+            var currentTrackId = manager.CurrentTrackId;
+            if (tracks == null || track.Name.Contains("*"))
+                return;
+            var found = false;
+            var index = 0;
+            foreach (var trackDescription in tracks)
+            {
+                index += 1;
+                if (GetTrackName(trackDescription, currentTrackId) == track.Name)
+                {
+                    found = true;
+                    manager.CurrentTrackId = trackDescription.Id;
+                    UpdateTracksListviewItemsSource(track, tracksListview);
+                    break;
+                }
+            }
+            if (!found)
+            {
+                manager.CurrentTrackId = -1;
+            }
+        }
+
+        private void UpdateTracksListviewItemsSource(TrackDescription selectedTrack, ListView? trackListView)
+        {
+            if (trackListView != null)
+            {
+                var itemSources = (ObservableCollection<TrackDescription>)trackListView.ItemsSource;
+                var previousTrack = itemSources.FirstOrDefault(t => t.Name[t.Name.Length - 1] == '*');
+
+                // Put "*" at the end of the selected track
+                itemSources.RemoveAt(selectedTrack.Id);
+                itemSources.Insert(selectedTrack.Id, new TrackDescription(selectedTrack.Id, $"{selectedTrack.Name} *"));
```

Comment:
I'm not a huge fan of replacing the item in the collection for the purpose of adding an asterisk for the selected track.

The asterisk should have been a converter or something, to allow different styles to show the current track the way they want.

Reply:
see https://github.com/videolan/libvlcsharp/pull/225#issuecomment-887233659

Reply:
PR updated.